config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml

```diff
@@ -904,7 +904,7 @@ presubmits:
       testgrid-tab-name: pull-perf-tests-verify-lint
     spec:
       containers:
-      - image: golangci/golangci-lint:v1.51.2
+      - image: golangci/golangci-lint:v1.58.2
```

Comment:
latest seems to be 1.59.1 https://github.com/golangci/golangci-lint/releases

Reply:
It seems that every minor version has two patches, so I took the last that already has two patches.
Not sophisticated logic behind that, but I assume the second patch is better than the first :)